Code/src/main/java/nl/utwente/group10/haskell/env/Env.java

```diff
@@ -23,9 +23,17 @@
      */
     private HashMultimap<Type, TypeClass> typeClasses;
 
+    /**
+     * @param exprTypes Map of Expr types.
+     * @param typeClasses Multimap of type classes.
+     */
+    public Env(Map<String, Type> exprTypes, HashMultimap<Type, TypeClass> typeClasses) {
+        this.exprTypes = exprTypes;
+        this.typeClasses = typeClasses;
+    }
+
     public Env() {
-        this.exprTypes = new HashMap<String, Type>();
-        this.typeClasses = HashMultimap.create();
+        this(new HashMap<String, Type>(), HashMultimap.create());
```

Comment:
No. The constructors of Env are used in other packages (catalog for example) and tests where an empty Env is needed.